interview-questions.md

```diff
@@ -10,46 +10,47 @@ Type your initials here to confirm you read the instructions:
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is a variable of the function and an argument is the value that is passed through the function.
 
-Researched answer:
+Researched answer: both are part of a function, a parameter is the name of the variable that is used to refer to the data that is passed through it, an argument is the literal data that is passed through the function
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-Your answer:
+Your answer: All I know is that it must take a function as a parameter...
 
-Researched answer:
+Researched answer: .map must take a function as a first or only parameter, the function will be applied to every element in the array, the second parameter is only needed when the function uses the "this" keyword. the first parameter is required but the second is only optional
```

Comment:
`.map` can take value, index, and array and parameters. The only one that is required is value.